docs/source/architecture/journal.rst

```diff
@@ -2,476 +2,196 @@
-   checked to see who is allowed to submit transactions and batches.
-
-#. **On-chain Block Validation Rules** - The on-chain block validation rules
-   are checked to ensure that the Block doesn't invalidate any of the
-   rules stored at ``sawtooth.validator.block_validation_rules``.
-
-#. **Batches Validation** - All of the Batches in the block are sent in order
-   to a Transaction Scheduler for validation. If any Batches fail validation,
-   this block is marked as invalid. Note: Batch and Signature verification is
-   done on receipt of the Batch prior to it being routed to the Journal. The
-   batches are checked for the following:
-
-    * No duplicate Batches
-    * No duplicate Transactions
-    * Valid Transaction dependencies
-    * Successful Batch Execution
-
-#. **Consensus Verification** - The Consensus instance is given to the Block for
-   verification. Consensus block verification is done by the consensus algorithm
-   using its own rules.
-
-#. **State Hash Check** - The StateRootHash generated by validating the block is
-   checked against the StateRootHash (state_root_hash field in the BlockHeader)
-   on the block. They must match for the block to be valid.
-
-If the block is computed to be valid, then StateRootHash is committed to the
-store.
 
 The BlockPublisher
 ==================
 
 The BlockPublisher is responsible for creating candidate blocks to extend the
-current chain. The BlockPublisher does all of the housekeeping work around
-creating a block but takes direction from the consensus algorithm for when to
-create a block and when to publish a block.
+current chain. The BlockPublisher does all of the housekeeping work for creating
+a block, but takes direction from the consensus algorithm for when to create a
+block and when to publish a block.
 
-The BlockPublisher follows this logic flow:
+The BlockPublisher responds to the following events:
 
-.. image:: ../images/journal_block_publisher_flow.*
-   :width: 80%
-   :align: center
-   :alt: Journal Block Publisher Diagram
-
-At each processing stage, the consensus algorithm has a chance to inspect and
-confirm the validity of the block.
+- Start block
+- Receive batch
+- Summarize block (stop and make the block available)
+- Finalize block (publish the block)
 
-During CreateBlock, an instance of Consensus.BlockPublisher is created that is
-responsible for guiding the creation of this candidate block.  Also, a
-TransactionScheduler is created and all of the pending Batches are submitted to
-it.
 
-A delay is employed in the checking loop to ensure that there is time for the
-batch processing to occur.
+The ChainController
+===================
 
-Genesis Operation
-=================
+The ChainController is responsible for maintaining and extending the
+blockchain for the validator. This responsibility involves validating proposed
+blocks, evaluating valid blocks to determine if they are the correct chain
+head, and generating new blocks to extend the chain.
```

Comment:
The ChainController does not "generating new blocks to extend the chain". Also, consensus handles "evaluating valid blocks to determine if they *should be" the new chain head" and the chain controller just handles making it so if consensus decides it should be.